### PR TITLE
GitHub Actions

### PR DESCRIPTION
**Pull request by DarqueWarrior**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**

- [x] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [x] Commands include options for Linux, MacOS, and Windows within codetabs
- [x] New file and folder names are globally unique
- [x] Page references use shortcodes instead of markdown or URL links
- [x] Images use HTML style and have alternative text
- [x] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

I was asked by Aaron to add GitHub Action integration page.

## Issue reference

#1634

### CODE REVIEW

File: daprdocs/content/en/developing-applications/integrations/github-actions.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+type: docs
+weight: 10000
+title: "GitHub Actions"
```
Comment (AaronCrawfis) [inline daprdocs/content/en/developing-applications/integrations/github-actions.md]:
```suggestion
title: "Use the Dapr CLI in a GitHub Actions workflow"
```

File: daprdocs/content/en/developing-applications/integrations/github-actions.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+linkTitle: "GitHub Actions"
+description: "Deploy your application with GitHub Actions"
```
Comment (AaronCrawfis) [inline daprdocs/content/en/developing-applications/integrations/github-actions.md]:
```suggestion
description: "Learn how to add the Dapr CLI to your GitHub Actions to deploy and manage Dapr in your environments."
```

File: daprdocs/content/en/developing-applications/integrations/github-actions.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+---
+
+Dapr can be integrated with GitHub Actions via the [Dapr tool installer](https://github.com/marketplace/actions/dapr-tool-installer) available in the GitHub Marketplace.
```
Comment (AaronCrawfis) [inline daprdocs/content/en/developing-applications/integrations/github-actions.md]:
```suggestion
Dapr can be integrated with GitHub Actions via the [Dapr tool installer](https://github.com/marketplace/actions/dapr-tool-installer) available in the GitHub Marketplace. This installer adds the Dapr CLI to your workflow, allowing you to deploy, manage, and upgrade Dapr across your environments.
```

File: daprdocs/content/en/developing-applications/integrations/github-actions.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+## Overview
+
+Before Dapr can be used in your Action Workflow you must install the Dapr CLI on the runner. Using the Dapr tool installer action will install the specified version of the Dapr CLI on macOS, Linux and Windows runners.
```
Comment (AaronCrawfis) [inline daprdocs/content/en/developing-applications/integrations/github-actions.md]:
```suggestion
The `dapr/setup-dapr` action will install the specified version of the Dapr CLI on macOS, Linux and Windows runners. Once installed, you can run any [Dapr CLI command]({{< ref cli >}}) to manage your Dapr environments.
```

Comment (AaronCrawfis) [reply daprdocs/content/en/developing-applications/integrations/github-actions.md]: Also add in the example you have from https://github.com/marketplace/actions/dapr-tool-installer to show the syntax with optional parameters.

File: daprdocs/content/en/developing-applications/integrations/github-actions.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+## Example
+
+```yaml
+- name: Install Dapr
+  uses: dapr/setup-dapr@v1
```
Comment (AaronCrawfis) [inline daprdocs/content/en/developing-applications/integrations/github-actions.md:19]:
```suggestion
  uses: dapr/setup-dapr@v1
  with:
    version: '1.3.0'
```

**Review by AaronCrawfis — CHANGES_REQUESTED** [review]

File: .github/workflows/website-v1-4.yml [hunk 6]
```diff
@@ -1,13 +1,13 @@
-name: Azure Static Web App v1.3
+name: Azure Static Web App v1.4
 
 on:
   push:
     branches:
-      - v1.3
```
Comment (greenie-msft) [inline .github/workflows/website-v1-4.yml:6]: @DarqueWarrior Should these workflow and config files be included in your changes?

**Comment by orizohar** [comment]
@DarqueWarrior looks like this PR includes unrelated changes, most likely due to a need to rebase (was your branch based on v1.4 and now merging into v1.3?). Can you please make sure only the new content is included?

**Comment by DarqueWarrior** [comment]
I tried to rebase this off of 1.3 after @AaronCrawfis  mentioned it. It should be corrected I think.

**Comment by DarqueWarrior** [comment]
Let me see if I can get this fixed up. I still see to many changes in this branch.

**Comment by greenie-msft** [comment]
@DarqueWarrior, another option is to create a new branch/PR from latest v1.3 branch and copy your changes over. Might be easier than trying to troubleshoot the current PR.

**Comment by greenie-msft** [comment]
I'm taking this PR over. Closing this PR and creating a new one.

**Pull request closed without merging**